packages/nimble-components/src/table/specs/table-column-specs/table-column-file-size.md

```diff
@@ -0,0 +1,294 @@
+# File Size Table Column
+
+## Overview
+
+The file size table column will provide a way to visualize numeric data that represents a file size (or other binary data size). It will automatically convert the raw value to use an appropriate display unit.
+
+### Background
+
+[Work Item](https://ni.visualstudio.com/DevCentral/_workitems/edit/2526114)
+
+### Non-goals
+
+-   Configurable number of decimal places
+-   Configurable units (e.g. display all values in KB)
+-   Grouping by value ranges/buckets
+
+### Features
+
+-   Converts the byte value to largest unit (e.g. KB, MB, GB) that results in a value of >=1
+-   Option to use 1024-based units (e.g. KiB) or 1000-based units (e.g. KB)
+-   Displays up to one decimal place
```

Comment:
Any reason why we aren't doing 2? "Quarters" are a fairly common mental unit that seems like it would be good to represent.

Reply:
You mean two decimal places? By "Quarters," are you saying we should be able to show "1.25"? I doubt the usefulness of showing precision to the hundreths place. SLE currently only shows up to one decimal place (though that's not a definitive argument). I'll leave this open to see if others have an opinion.

Reply:
Yes, sorry, two decimal places. The usefulness, to me, is intrinsic in that .25 is a half of a half, which isn't an uncommon value to represent, particularly when dealing in concepts that are powers of 2. I would think seeing something like 1.3 MiB is odder than seeing something like 1.25 MiB.

Reply:
I'll also point out that the existing SLE code shows no decimal place for bytes and a maximum of 1 decimal place for everything else.

Reply:
Sounds like we're okay with up to one decimal place.